src/hotspot/share/code/codeBlob.hpp

```diff
@@ -511,6 +547,16 @@ class DeoptimizationBlob: public SingletonBlob {
   }
   address implicit_exception_uncommon_trap() const { return code_begin() + _implicit_exception_uncommon_trap_offset; }
 #endif // INCLUDE_JVMCI
+
+  void print_value_on(outputStream* st) const;
+
+  class Vptr : public CodeBlob::Vptr {
```

Comment:
This one specifically
```suggestion
  class Vptr : public SingletonBlob::Vptr {
```